src/utils/nextRunAt.ts

```diff
@@ -28,21 +29,30 @@ export function isValidHumanInterval(value: unknown): value is string {
 export const computeFromInterval = (attrs: IJobParameters): Date => {
 	const previousNextRunAt = attrs.nextRunAt || new Date();
 	log('[%s:%s] computing next run via interval [%s]', attrs.name, attrs._id, attrs.repeatInterval);
+
 	const lastRun = dateForTimezone(attrs.lastRunAt || new Date(), attrs.repeatTimezone);
+
+	const cronOptions: ArgumentsType<typeof parseExpression>[1] = {
+		currentDate: lastRun.toDate()
+	};
+
+	if (attrs.repeatTimezone) {
```

Comment:
if timezone is not set, it should be therefore undefined, and setting cronOptions.tz = undefined shoul dbe equivalnt, if types allow this, I would remove this if, and set the timezone directly in like 36 (like currentDate)